app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt

```diff
@@ -117,6 +124,15 @@ class DetailArtObjectActivity : AppCompatActivity() {
         }
     }
 
+    private fun shareArtObjectInfo() {
+        val shareIntent = Intent()
+        shareIntent.action = Intent.ACTION_SEND
+        shareIntent.putExtra(Intent.EXTRA_TEXT,
+                "${artObjectUi!!.name}  - (${artObjectUi!!.lat}, ${artObjectUi!!.lng})")
+        shareIntent.type = "text/plain"
+        startActivity(Intent.createChooser(shareIntent, resources.getText(R.string.share)))
```

Comment:
We should also share the image.
The most common use case: is to share image on instagram.